tools/ci_build/requirements/requirements-pybind.txt

```diff
@@ -0,0 +1,9 @@
+setuptools
+wheel
+pytest
+numpy==1.24.0 ; python_version < '3.12'
+numpy==1.26.0 ; python_version >= '3.12'
```

Comment:
I have a PR(#21085) that updates Numpy to a newer version. Could you wait it? I think we have conflicts here.

Reply:
update to `numpy >= 1.19.0`, same as your PR.